snippets/card-product.liquid

```diff
@@ -482,7 +482,7 @@
                           {% endif %}
                           class="link product__view-details animate-arrow small-hide medium-hide"
                         >
-                          {{ 'products.product.view_full_details' | t }}
+                          {{ 'products.product.view_full_details' | t -}}
                           {% render 'icon-arrow' %}
```

Comment:
Now I wonder if this is redundant 🤔 Since we can already click on the product title to get us there. 
What do you think ?